.changeset/sour-fishes-talk.md

```diff
@@ -0,0 +1,5 @@
+---
+'@nhost/dashboard': minor
+---
+
+don't show deprecated plans
```

Comment:
I think it's better to add a `chore: ` or `feat: ` to this so it's consistent with the rest of the changelog

Reply:
fixed